pkg/api/login_oauth.go

```diff
@@ -97,9 +97,7 @@ func (hs *HTTPServer) OAuthLogin(ctx *models.ReqContext) {
 
 	code := ctx.Query("code")
 	if code == "" {
-		// FIXME: access_type is a Google OAuth2 specific thing, consider refactoring this and moving to google_oauth.go
-		opts := []oauth2.AuthCodeOption{oauth2.AccessTypeOffline}
-
+		var opts []oauth2.AuthCodeOption
```

Comment:
With these last changes, the extra `AuthCodeOption`s that were added to gitlab,github and azure will be removed regardless of the feature flag.

From our discussion I think that's fine since it's not used by these providers but it's a small increase in risk compared to the previous version that we should be aware of